.github/workflows/deploy.yml

```diff
@@ -8,7 +8,7 @@ on:
       - "!skipci*"
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref_name }}
```

Comment:
github.ref is the same as GITHUB_REF and fully qualifies the path, so a branch `berrydidthis` would appear as `/refs/heads/berrydidthis` hence the need to parse out the strings.

In destroy, GITHUB_REF and github.ref will always point to `master` which is the default branch on which destroy is executed.  In that case, we need to get the reference from the event.

What annoys me is that github.ref_name and github.event.ref are both just the branch without the full qualification.  I would prefer a consistent naming convention but this is out of my hands.